src/content/docs/en/guides/backend/google-firebase.mdx

```diff
@@ -219,14 +219,18 @@ export const GET: APIRoute = async ({ request, cookies, redirect }) => {
     expiresIn: fiveDays,
   });
 
-  cookies.set("session", sessionCookie, {
+  cookies.set("__session", sessionCookie, {
     path: "/",
   });
 
   return redirect("/dashboard");
 };
 ```
 
+:::caution
+Cookie must be called '__session'. Firebase functions [will remove all cookies except for that one](https://firebase.google.com/docs/hosting/manage-cache#using_cookies).
```

Comment:
To me, this reads like "a session cookie must have this name" when, in fact, it is the only cookie allowed _at all_. 

I think the warning should be a little stronger.

```suggestion
Firebase only allows the use of [one cookie, and it must be named `__session`](https://firebase.google.com/docs/hosting/manage-cache#using_cookies). Any other cookies the client sends will not be visible to your application.
```